Homework-№4-from-28.09.2022/task 1.c

```diff
@@ -0,0 +1,183 @@
+#include <stdio.h>
+#include <stdlib.h>
+#include <stdbool.h>
+#define size 32 //in this task all lengths are a 32
```

Comment:
```suggestion
#define SIZE 32 //in this task all lengths are a 32
```
Имена define констант принято капсом писать